generator.rb

```diff
@@ -0,0 +1,56 @@
+puts "Hello, welcome to the Portmanteau machine"
+
+def is_vowel?(letter)
+  if letter =~ /[aeiou]/
+    return true
+  elsif letter !~ /[aeiou]/
```

Comment:
Nice use of a regular expression here! We'll talk about those more formally in a few weeks.